site/src/pages/CreateWorkspacePage/CreateWorkspacePage.tsx

```diff
@@ -65,7 +65,16 @@ const CreateWorkspacePage: FC = () => {
 	});
 	const permissionsQuery = useQuery({
 		...checkAuthorization({
-			checks: createWorkspaceChecks(templateQuery.data?.organization_id ?? ""),
+			checks: {
+				...createWorkspaceChecks(templateQuery.data?.organization_id ?? ""),
+				canUpdateTemplate: {
+					object: {
+						resource_type: "template",
+						resource_id: templateQuery.data?.id ?? "",
+					},
+					action: "update",
+				},
```

Comment:
you should just add this inside of the `createWorkspaceChecks` helper, that's what it is for